### ISSUE TITLE
Can you crack with method nextInt(bound) ???

### ISSUE THREAD

**Issue by jasonx8891** [sacundim/cracking-prngs#1]
I'm try to find out the ways to crack method nextInt(bound), but it too difficult, have you try this ?

**Comment by SilentSword123456** [comment]
yeah, me too, @ldcasillas-progreso, can you try adding that function please?
